packs/data/other-effects.db/effect-2-circumstance-penalty-to-spell-attack-rolls-and-spell-dcs.json

```
@@ -0,0 +1,54 @@
+{
+    "_id": "BjkhK1nFUw3vNKg2",
+    "img": "systems/pf2e/icons/effects/crtical-hit-fumble.webp",
+    "name": "Effect: -2 circumstance penalty to spell attack rolls and spell DC's",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p><strong>Effect applied by Critical Hit/Fumble Deck<br /></strong>-2 circumstance penalty to spell attack rolls and spell DC's</p>"
+        },
+        "duration": {
+            "expiry": "turn-end",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "circumstance penalty to spell attacks",
+                "selector": "spell-attack",
+                "type": "circumstance",
+                "value": -2
+            },
+            {
+                "key": "FlatModifier",
+                "label": "circumstance penalty to spell DC's",
+                "selector": "spell-dc",
+                "type": "circumstance",
+                "value": -2
+            }
```

```suggestion
            {
                "key": "FlatModifier",
                "label": "circumstance penalty to spell attacks",
                "selector": [
                    "spell-attack",
                    "spell-dc"
                ],
                "type": "circumstance",
                "value": -2
            }
```
We can condense rule elements by using selector arrays.